src/content/users/ProfileView.tsx

```diff
@@ -601,21 +603,30 @@ const ProfileView: FC<Props> = ({ _id, viewType }: Props) => {
                       control={control}
                       rules={{ required: false }}
                       render={({ field }) => (
-                        <StyledSelect
+                        <StyledAutocomplete
                           {...field}
-                          size="small"
-                          value={field.value || ""}
+                          options={institutions}
+                          getOptionLabel={(option: Institution) => option.name}
+                          isOptionEqualToValue={(option: Institution, value: Institution) =>
+                            option._id === value._id
+                          }
+                          onChange={(_, data: Institution | null) =>
+                            field.onChange(data?._id || "")
+                          }
+                          value={institutions.find((i) => i._id === field.value) || null}
+                          renderInput={({ inputProps, ...params }) => (
+                            <TextField
+                              {...params}
+                              inputProps={{
+                                "aria-labelledby": "userInstitution",
+                                "data-testid": "institution-input",
+                                ...inputProps,
+                              }}
+                              placeholder="Select an Institution"
+                            />
+                          )}
                           disabled={fieldset.institution === "DISABLED"}
```

Comment:
As much as I dislike horizontal scrollbars, we probably should use one instead of line wrapping long institution names (KC preference).

The studies dropdown uses these papers which should solve it, unless you have an alternative.

(current)

<img width="369" height="354" alt="image" src="https://github.com/user-attachments/assets/7de3b5a2-649c-442c-b886-0d0e384fdbe6" />

(after)

<img width="369" height="354" alt="image" src="https://github.com/user-attachments/assets/48c56501-31ab-46b6-9be0-11af7ef244e8" />

```suggestion
                          disabled={fieldset.institution === "DISABLED"}
                          PaperComponent={StyledPaper}
                          PopperComponent={StyledPopper}                          
```